third-party/chpl-venv/test-requirements.txt

```diff
@@ -1,4 +1,3 @@
-argparse==1.3.0
-PyYAML==3.13
-filelock==2.0.13
-argcomplete==1.9.4
+PyYAML==5.3.1
+filelock==3.0.12
+argcomplete==1.12.1
```

Comment:
Part of the reason we pinned to older versions was to stick with ones that worked with python2. Now that we're using 3, I wonder if we can/should just use the latest version. 

I'm fine with pinning to specific versions that we know work too, but wouldn't be opposed to using whatever the default version is.

Reply:
Pinning to specific versions means we don't have to worry about a package update coming in when we aren't prepared to deal with it (like, post code freeze or whatever).  I've set myself a reminder every six months to look at if we need to update versions.